gitbutler-ui/src/routes/settings/+page.svelte

```diff
@@ -176,10 +176,32 @@
 			{/if}
 
 			<SectionCard>
-				<svelte:fragment slot="title">Appearance</svelte:fragment>
+				<svelte:fragment slot="title">Theme</svelte:fragment>
 				<ThemeSelector {userSettings} />
 			</SectionCard>
 
+			<SectionCard orientation="row" centerAlign>
+				<svelte:fragment slot="title">Hunks tab size</svelte:fragment>
+
+				<svelte:fragment slot="actions">
+					<TextBox
+						type="number"
+						width={100}
+						textAlign="center"
+						value={$userSettings.tabSize.toString()}
+						minVal={2}
```

Comment:
I'd imagine some people prefer a tab size of 1, so maybe that should be the minimum?

Reply:
I went ahead and made this change, so please let me know if you disagree

Reply:
All good. Thanks, didn't know about this